src/FixedDataTable.js

```diff
@@ -817,6 +820,7 @@ var FixedDataTable = createReactClass({
 
   _renderRows(/*number*/ offsetTop) /*object*/ {
     var state = this.state;
+    var showScrollbarY = state.maxScrollY > 0 && state.overflowY !== 'hidden' && state.showScrollbarY !== false;
```

Comment:
this is duplicated above.  Should we pull out to a helper?

Reply:
Yeah that's a good call. Do we want that to actually be in FixedDataTableScrollHelper.js, or just as an additional method here in FixedDataTable.js?

Reply:
I think leaving a private method here is fine since it's only used in this file.